src/Anemone/Foreign/Mempool.hs

```diff
@@ -47,6 +48,9 @@ foreign import ccall unsafe "anemone_mempool_free"
 foreign import ccall unsafe "anemone_mempool_total_alloc_size"
   totalAllocSize :: Mempool -> IO Int64
 
+foreign import ccall unsafe "anemone_is_pointer_aligned"
+  isPointerAligned :: Ptr Void -> IO Bool
```

Comment:
This should be [CBool](https://github.com/ambiata/anemone/blob/master/src/Anemone/Foreign/Data.hs#L20) as you used `bool_t` typedef in the C signature